implant/sliver/handlers/handlers.go

```diff
@@ -614,12 +614,58 @@ func gzipRead(data []byte) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
+func standarizeArchiveFileName(path string) string {
```

Comment:
Fixing issues with how names of files are generated in tar archives so that names are stored relative to the filesystem root.

Reply:
Now that I think about it, we might want to check for symbolic links and decide whether we follow them or not. We wouldn't want an extracted archive to overwrite a local file on the operator's system.

Reply:
I do not think they would because this change makes it so that the file paths are stored in the archive relative to the volume root. For example, if you had a symlink to `/etc/passwd` in `/tmp/passwd`, it would be stored in the archive as `tmp/passwd` (no leading `/`).